### ISSUE TITLE
New lines not converted

### ISSUE THREAD

**Issue by gpapin** [hipchat/hipchat-cli#27]
When sending message with new lines ( `\n` ) the content is not completely cleared resulting on non valid json send, and you get this message back : 

```
{
  "error": {
    "code": 400,
    "message": "The request body cannot be parsed as valid JSON: Invalid control character at: line 1 column 121 (char 121)",
    "type": "Bad Request"
  }
}
```

Here is a quick fix. Replace this block line 100 

```
# replace newlines with XHTML <br>
if [ $FORMAT == 'html' ]; then
    INPUT=$(echo -n "${INPUT}" | sed "s/$/\<br\>/")
    INPUT=$(echo -n "${INPUT}" | tr '\n' ' ')
fi
```

**Comment by sebaplaza** [comment]
Alternative fix:
INPUT=$(echo "${INPUT}" | sed 's/$/<br>/' |  tr -d '\n\r')

**Comment by JeffChien** [comment]
another fix with awk
```sh
INPUT=$(echo -n "${INPUT}" | awk NF=NF RS= OFS='<br>')
```

**Comment by jimsmith** [comment]
Another one almost a year old, this should be progressed with.

**Comment by dkolbly** [comment]
Fixed in #30
